src/popup/plugins/veeValidate.ts

```diff
@@ -114,6 +120,25 @@ defineRule(
   },
 );
 
+defineRule(
+  'address_book_entry_exists',
+  (value: string, [entries, savedKey, type]: [IAddressBook, string, 'name' | 'address']) => {
+    const key = (Object.keys(entries) as Array<AccountAddress>)
+      .find((k) => entries[k][type] === value);
+
+    const entryTypeText = type === 'name' ? tg('common.name') : tg('common.address');
+
+    return (key && key !== savedKey)
+      ? tg('validation.addressBookEntryExists', [entryTypeText.toLowerCase()])
+      : true;
+  },
+);
+
+defineRule(
+  'is_address_valid',
+  (value: string) => getProtocolByAddress(value) ? true : tg('validation.invalidAddress'),
+);
```

Comment:
We have similar rule: `account_address`.

Reply:
`account_address` validates for a given protocol and network type where `is_address_valid` will check if the address is valid in any of the supported protocols/networks